stack.yaml.lock

```diff
@@ -0,0 +1,33 @@
+# This file was autogenerated by Stack.
```

Comment:
Is this file supposed to be checked in?

Reply:
According to https://docs.haskellstack.org/en/stable/lock_files/#lock-files it can be checked into source control, but it's not strictly necessary if we don't want to include it. I'm fine with either way.